lib/DDG/Spice/NewYorkTimes.pm

```diff
@@ -0,0 +1,22 @@
+package DDG::Spice::NewYorkTimes;
+# ABSTRACT: Details on New York Times best sellers
+
+use strict;
+use DDG::Spice;
+
+spice to => 'https://api.nytimes.com/svc/books/v3/lists/best-sellers/history.json?api-key={{ENV{DDG_SPICE_NewYorkTimes_APIKEY}}}&title=$1';
+
+triggers startend => "new york times", "new york times best seller";
```

Comment:
We're going to need more niche trigger for this. That majority of queries containing "new york times" are not going to be a search for a book review:
![drake_at_duckduckgo](https://cloud.githubusercontent.com/assets/873785/15654707/7574b3c2-2664-11e6-8ee2-ca9c450a5ea5.png)


It looks like "new york times review", "new york times book review", and "new york times review of" are pretty good though:

![drake_at_duckduckgo](https://cloud.githubusercontent.com/assets/873785/15654700/60e4f250-2664-11e6-8da7-126c0e23273c.png)